src/app/pages/contact/contact.js

```diff
@@ -48,6 +48,7 @@ export default class Contact extends SalesforceForm {
         };
         this.model = {
             salesforceHome: salesforce.salesforceHome,
+            salesforceOid: salesforce.oid,
```

Comment:
If the settings suggestion above is used then I think this would just become `settings.SALESFORCE_OID` or something similar (and easy to grep to find all the occurrences of)

Reply:
I could give the models a `salesforce` member that is just the imported `salesforce` data. The template would then use `model.salesforce.salesforceHome` and `model.salesforce.oid`.